src/main/java/com/puppycrawl/tools/checkstyle/utils/CheckUtil.java

```diff
@@ -680,4 +680,14 @@ public static String getShortNameOfAnonInnerClass(DetailAST literalNewAst) {
         final DetailAST firstChild = parentAst.getFirstChild();
         return extractQualifiedName(firstChild);
     }
+
+    /**
+     * Checks if the given file name is a package-info.java file.
+     *
+     * @param fileName name of the file.
+     * @return true if the package file.
+     */
+    public static boolean inPackageInfo(String fileName) {
```

Comment:
lets name it `isPackageInfo`  , preffix "in" confuse me.

Reply:
done